README.md

```diff
@@ -44,10 +44,10 @@ Prerequisites:
 - [VSCode](https://code.visualstudio.com/download)* *optional*
 
 Instructions:
-1. In this Github repository, click on the Code dropdown, select Codespace and click `Create`. 
+1. In this Github repository, click on the Code dropdown, select Codespace and click `Create`.
 2. Confirm codespace settings, for Machine type select `2-core` (this can be changed later if you require more power)
 
-This will setup a Dockeer container to run in Github with all the necessary dependencies (python, pip, git, ...) with the correct versions. An active internet connection will be required for this, and this will use up your monthly allowance for Github codespace. 
+This will setup a Dockeer container to run in Github with all the necessary dependencies (python, pip, git, ...) with the correct versions. An active internet connection will be required for this, and this will use up your monthly allowance for Github codespace.
```

Comment:
Fine with adding how to terminate but would discourage adding how much allowance you get because:
- this can easily be checked by the user
- GitHub may change these at any time
- user may have team/organisation settings which may cause their codespace allowance to differ

Reply:
Good shout - agree with you @KYLChiu